src/engine/experimental/blob-extension.md

```diff
@@ -54,7 +54,6 @@ This structure has the syntax of `ExecutionPayloadV2` and appends a single field
 
 The fields are encoded as follows:
 
-- `blockHash`: `DATA`, 32 Bytes
 - `kzgs`: `Array of DATA` - Array of `KZGCommitment` as defined in [EIP-4844](https://eips.ethereum.org/EIPS/eip-4844), 48 bytes each (`DATA`).
```

Comment:
```suggestion
- `commitments`: `Array of DATA` - Array of `KZGCommitment` as defined in [EIP-4844](https://eips.ethereum.org/EIPS/eip-4844), 48 bytes each (`DATA`).
```
Since kzgs are being renamed to the more generic `commitments` in https://github.com/ethereum/EIPs/pull/6610 it would be nice to rename it here as well in single shot